opm/parser/eclipse/EclipseState/Schedule/Group/GuideRate.hpp

```diff
@@ -101,13 +99,17 @@ struct GRValState {
     bool has(const std::string& name) const;
     bool has(const std::string& name, const Phase& phase) const;
 
+    void update_time_stamp(double sim_time, size_t report_step);
```

Comment:
Should just be updated inline!

Reply:
The time stamp can only be updated after all the wells and groups finished update. Referring to the discussion here, 
https://github.com/OPM/opm-common/pull/2509#discussion_r655956230